src/modules/tas_studio/editor/mod.rs

```diff
@@ -1418,6 +1436,52 @@ impl Editor {
         Ok(())
     }
 
+    fn tick_side_strafe_yawspeed_adjustment(
+        &mut self,
+        mouse: MouseState,
+        keyboard: KeyboardState,
+    ) -> eyre::Result<()> {
+        let Some(adjustment) = &mut self.side_strafe_yawspeed_adjustment else {
+            return Ok(());
+        };
+
+        let bulk_idx = self.selected_bulk_idx.unwrap();
+        let (bulk, first_frame_idx) =
+            bulk_and_first_frame_idx_mut(&mut self.branches[self.branch_idx].branch.script)
+                .nth(bulk_idx)
+                .unwrap();
+
+        let yawspeed = bulk.constant_yawspeed_mut().unwrap();
+
+        if !mouse.buttons.is_right_down() {
+            if !adjustment.changed_once {
+                self.side_strafe_yawspeed_adjustment = None;
+                return Ok(());
+            }
+
+            let op = Operation::SetYawspeed {
+                bulk_idx,
+                from: adjustment.original_value,
+                to: *yawspeed,
+            };
+            self.side_strafe_yawspeed_adjustment = None;
+            return self.store_operation(op);
+        }
+
+        // delta scalar might need being a lot higher but there's +ALT1 adjustment anyway.
+        let speed = keyboard.adjustment_speed();
+        let delta = adjustment.delta(mouse.pos.as_vec2()) * 0.1 * speed;
+        let new_yawspeed = adjustment.original_value + delta;
```

Comment:
Should this be maxed with 0? This brings another question, what happens if the yawspeed is negative? Does it even make sense?

Reply:
Yes, yawspeed can be negative and it strafes funny. I just don't want to deal with handling sign and extra mechanism so I allow yawspeed to be anything it wants. 0 is also a valid yawspeed.